collections/_infographics/emissions/fossil-fuels-emissions.md

```diff
@@ -1,16 +1,60 @@
 ---
 layout:        infographic
-title:         "Global CO2 emissions: fossil fuels & cement production"
+title:         "Global CO₂ emissions: fossil fuels & cement production"
 slug:          "fossil-fuels-emissions"
 redirect_from: "/fossil-fuels-emissions"
-published:     2022-07-26
+published:     2022-11-22
 weight:        95
 tags-scopes:   [ world ]
 tags-topics:   [ emissions, energy ]
-caption:       "Global CO2 emissions from coal, crude oil and natural gas combustion have increased fourfold over the past 60 years. These emissions will have to be reduced to almost zero if we are to achieve carbon neutrality."
+caption:       "Global CO<sub>2</sub> emissions from burning coal, oil and gas have quadrupled over the past 60 years. These emissions will have to be reduced to almost zero if we are to achieve carbon neutrality."
 data-our:      "https://docs.google.com/spreadsheets/d/19PiqYV6FkttzrNcjkbTQmphzxgydtTc1MB6nDZpybj0/edit?usp=sharing"
 data-orig:
   - [ "Global Carbon Budget", "https://www.icos-cp.eu/science-and-impact/global-carbon-budget/2020" ]
 ---
 
-{% include includes-local/comment-placeholder.md slug="infografiky/emise-fosilni-paliva" %}
+## How to read this graph?
+
+* The graph shows **worldwide emissions from burning coal, oil and natural gas** and, for comparison, **emissions from cement production**. Total emissions from these sources for 2019 were 35,897 million tons of CO<sub>2</sub>.
+* The graph only shows emissions from the chemical reaction CaCO<sub>3</sub> + heat ⟶ CaO + CO<sub>2</sub> as emissions from cement production. Emissions created by burning coal or gas for heating are counted as emissions of the respective fuel.
+
+## More about individual fuels and cement
+
+{% comment %}
+{% include preview-box.html
+    title="What is electricity produced from?"
+    text="Unlike oil, coal and natural gas are important fuels for electricity generation."
+    slug="elektrina-svet"
+%}
+{% endcomment %}
+
+### Coal
+
+* **Usage:** Primarily for the production of electricity and heat ([Mapped worlds coal power plants](https://www.carbonbrief.org/mapped-worlds-coal-power-plants)) in the iron or glass smelting industry or other heating.
```

Comment:
would change to [Map of world's coal power plants]

Reply:
Thanks, I corrected it to `[Map of world's coal power plants]`